<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1094</width>
    <height>562</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>IMU Calibration Tool</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="Renderer" name="openGLWidget"/>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1094</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionConnect"/>
   <addaction name="actionDisconnect"/>
   <addaction name="separator"/>
   <addaction name="actionCompass"/>
   <addaction name="actionCalibration"/>
   <addaction name="separator"/>
   <addaction name="actionDone"/>
   <addaction name="actionCancel"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionConnect">
   <property name="text">
    <string>Connect</string>
   </property>
  </action>
  <action name="actionDisconnect">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Disconnect</string>
   </property>
  </action>
  <action name="actionCalibration">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Calibrate</string>
   </property>
   <property name="iconText">
    <string>Calibrate</string>
   </property>
  </action>
  <action name="actionDone">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Done</string>
   </property>
   <property name="visible">
    <bool>true</bool>
   </property>
  </action>
  <action name="actionCancel">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Cancel</string>
   </property>
  </action>
  <action name="actionCompass">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Compass</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>Renderer</class>
   <extends>QOpenGLWidget</extends>
   <header>renderer.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
